app/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
 
 
+
```

Comment:
wy

Reply:
Hum, I added a dependency which doesn't work in the end, so I removed it but I forgot to delete the new line in the build.gradle ^^